sql/core/src/test/scala/org/apache/spark/sql/test/SQLTestUtils.scala

```diff
@@ -424,6 +425,14 @@ private[sql] trait SQLTestUtilsBase
   protected def testFile(fileName: String): String = {
     Thread.currentThread().getContextClassLoader.getResource(fileName).toString
   }
+
+  /**
+   * Returns the size of the local directory except the metadata file and the temporary file.
+   */
+  def getLocalDirSize(file: File): Long = {
+    assert(file.isDirectory)
+    file.listFiles.filter(f => DataSourceUtils.isDataFile(f.getName)).map(_.length).sum
```

Comment:
Nit: we can use `f.toPath` so that we don't need to modify `DataSourceUtils`

Reply:
It's `java.nio.file.Path`, not `org.apache.hadoop.fs.Path`:
![image](https://user-images.githubusercontent.com/5399861/60515505-543e9000-9d0e-11e9-863f-03f8a420bc19.png)